### ISSUE TITLE
Dark theme stop working propertly when ViewTransitions is used

### ISSUE THREAD

**Issue by jsparedes** [zeon-studio/astroplate#12]
After **ViewTransitions** in `Base.astro`

```astro
---
import { ViewTransitions } from 'astro:transitions';
---
<html lang="en">
  <head>
    <title>My Homepage</title>
    <ViewTransitions />
  </head>
  <body>
    
  </body>
</html>
```

If we switch to another page and want to change the theme (dark or ligth): It stop working. In order to change the theme it is necessary to do a reload. This means what we can only change the theme in the first page loaded.

**Comment by tfsomrat** [comment]
closed as duplicate #8

**Comment by tfsomrat** [comment]
hey @jsparedes, this issue is fixed here #16
